### PR TITLE
Copy local project git config to temporary drat dir

### LINKED ISSUE #21: Don't assume globally set user and email configuration in deploy_minicran_package()
I don't have this on my system: 

```r
withr::with_tempdir({ gert::git_init(); gert::git_signature_default() })
```

gives an error. Can we copy over the current repo's config to the temp repo?

### PR DESCRIPTION
**Pull request by pat-s**
fix #21

### CODE REVIEW

File: R/deploy.R [hunk 1]
```diff
@@ -60,6 +60,9 @@ deploy_minicran_package <- function(drat_repo,
   pkgname <- desc::desc_get("Package")
   pkgversion <- desc::desc_get_version()
 
+  # copy git config in case config is not set globally
+  file.copy(".git/config", paste0(drat_dir, "/.git/config"), overwrite = TRUE)
```
Comment (krlmlr) [inline R/deploy.R:64]:
```suggestion
  file.copy(".git/config", file.path(drat_dir, ".git/config"), overwrite = TRUE)
```

File: R/deploy.R [hunk 2]
```diff
@@ -60,6 +60,9 @@ deploy_minicran_package <- function(drat_repo,
...
+
   withr::with_dir(
```
Comment (krlmlr) [inline R/deploy.R:66]: Unrelated: `local_dir()` saves one level of indentation. Maybe it's worth splitting this function into smaller parts?

**Review by krlmlr — APPROVED** [review]
Looks good, thanks!

File: R/deploy.R (hunk 2 above)
Comment (pat-s) [reply R/deploy.R:66]: Not entirely sure how to apply this here, feel free to tweak.

**Pull request merged**